### PR TITLE
Update comments in vof_filter.h

### PR DESCRIPTION
**Pull request by AmishgaAlphonius**

### CODE REVIEW

File: include/solvers/vof_filter.h [hunk 1]
```diff
@@ -21,46 +21,61 @@
 #include <core/parameters_multiphysics.h>
 
 /**
- * @brief Class that allows phase fraction filtering
+ * @brief Abstract class for filtering phase fraction in volume of fluid (VOF)
+ * simulations.
```
Comment (blaisb) [inline include/solvers/vof_filter.h:25]: I would remove the abstract class. What do you think?

**Review by blaisb — APPROVED** [review]
One tiny change

File: include/solvers/vof_filter.h (hunk 1 above)
Comment (AmishgaAlphonius) [reply include/solvers/vof_filter.h:25]: I think that the **brief** should say that you can't directly instantiate an object of the class; should it not?

Comment (blaisb) [reply include/solvers/vof_filter.h:25]: Agreed

**Pull request merged**